docs/source/algorithms/SetMDFrame-v1.rst

```diff
@@ -30,11 +30,11 @@ Usage
 
 **Example - Sample usage of SetMDFrame**
 
-.. testcode:: SetMDFrameExample
+.. code-block:: python
```

Comment:
I'm not sure why this file is changed?

Reply:
Was getting systematic segfaults on building the documentation, I can revert this but there are genuine problems with this and the SofTwoThetaTOF doc tests.

Reply:
Ah ok. No problem, I just wanted to understand :)